base_app/README.md

```diff
@@ -1,62 +1,12 @@
-# NgSimpleUi
+# Base_app
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 7.1.1.
-
-The project was updated to `Angular 13.3` in several phases using the instructions in the [Angular Update Guide](https://update.angular.io/?l=3&v=7.2-13.0)
-with the following settings:
-
-**Angular Versions**  
-From: `7.2`, To: `8.0`  
-From: `7.2`, To: `9.0`  
-From: `7.2`, To: `10.0`  
-From: `7.2`, To: `11.0`  
-From: `7.2`, To: `13.0`
-
-**App Complexity** `Advanced`
-
-**Other Dependencies**
-- [ ] I use ngUpgrade to combine AngularJS & Angular
-- [x] I use Angular Material
-- [ ] I use Windows
-
-## Build
-
-Run `ng build-client-dev` to build a version of the `base-app` that contains
- `.map.js` files that allow direct debugging of `typescript` inside a
-browser debugger.
-
-Run `ng build-client-prod` to build an optimized version without the `.map` files
-that's intended for distribution.
-
-The build artifacts are stored in the `dist/` directory.
-
-
-## Development server
-
-Run `ng serve` to develop using a dev web server.
-Navigate to `http://localhost:4200/`.
-The app will automatically reload if you
-change any of the source files.
-
-Note that this mode of development isn't currently being used on a regular basis by the developers
-and may be broken between revisions.
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli/blob/master/README.md)
+[Angular Update Guide](https://update.angular.io/)
 
 ## Code scaffolding
 
 Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
-## Running unit tests
-
-See the [Readme for Unit Tests](./src/test/00.readme.unit-tests.md)
-
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
-
-## Running end-to-end tests
-
-See the [Readme for E2E Tests](./src/test/00.readme.e2e-tests.md)
-
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
-
```

Comment:
Are these no longer applicable?

Reply:
The previous information in those documents was old and irrelevant (EX: we don't have any e2e tests) . I added instructions on how to run our testing in [this section](https://github.com/greg-ses/simpleui/tree/improve-testing#testing) of the readme